src/js/components/Tabs/stories/Uncontrolled.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { Attraction, Car, TreeOption } from 'grommet-icons';
+import { Box, Grommet, Tab, Tabs } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const UncontrolledTabs = ({ plain }) => (
+  <Grommet theme={grommet} full>
+    <Box fill>
+      <Tabs flex>
+        <Tab plain={plain} title="Tab 1">
+          <Box fill pad="large" align="center" background="accent-1">
+            <Attraction size="xlarge" />
+          </Box>
+        </Tab>
+        <Tab plain={plain} title="Tab 2">
+          <Box fill pad="large" align="center" background="accent-2">
+            <TreeOption size="xlarge" />
+          </Box>
+        </Tab>
+        <Tab plain={plain} title="Tab 3">
+          <Box fill pad="large" align="center" background="accent-3">
+            <Car size="xlarge" />
+          </Box>
+        </Tab>
+      </Tabs>
+    </Box>
+  </Grommet>
+);
+
+UncontrolledTabs.defaultProps = {
+  plain: false,
```

Comment:
I think we don't need the UncontrolledTabs.defaultProps, and assign false on `plain` as part of the declaration on line #8.

Reply:
Done.